packages/terra-clinical-item-display/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Added new prop for icon alignment and missing default primary text style
```

Comment:
Lets break this into two bullets

```
### Added
- iconAlignment prop

### Fixed 
- Added missing primary text style

Reply:
Fixed: https://github.com/cerner/terra-clinical/pull/490/commits/c86aab122395aef6137dbf691f69476d0949ab57#diff-589055d761929f3cff8e25e6d51ef5b5